ci-operator/step-registry/origin/e2e/azure/origin-e2e-azure-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: origin-e2e-azure
+  steps:
+    pre:
+    - chain: ipi-azure-pre
+    test:
+    - ref: origin-e2e-test
+    post:
+    - chain: ipi-azure-post
```

Comment:
I'm ok with this because it is the current convention. But having all the ipi platforms create a separate chain for post which is ultimately just `ipi-deprovision` underneath seems like an unnecessary level of indirection.

Reply:
What 's different between the underlying implementations?

Reply:
Ah, right now it's just the deprovision chain. This approach lets you add some provider-specific thing but they *do* reuse the same deprovision code underneath